### DISCUSSION TITLE
When mouse is in the topmost area in screen, I can't click browser tabs

### DISCUSSION THREAD

**Discussion by developer1105** [i3/i3#4994] [Q&A] [answered]
In Firefox, and Chrome, when the mouse is in topmost region, I can't click on tabs. I need to move the cursor down some pixels, so it can work. I remember that this does not happen in other wms (like Openbox, for instance). What can this be? Is there a config or workaround for this?

**Comment by Airblader** [comment]
Do you have window border enabled? I would guess your mouse is on the border, not on the tab.

**Reply by developer1105** [reply]
It seems the issue is browser-related. With some browsers "border none" solves the issue. With others "fullscreen enabled" is needed, and for some others, none of both configs work.
Well, probably there is no universal solution for it, and I did a stupid bug report. Sorry.

**Reply by developer1105** [reply, marked as answer]
This seems to be a GTK issue. Didn't have a clue about it, back then.

**Reply by taylankasap** [reply]
@developer1105 Were you able to solve this on Chrome with classic theme
